packages/graphback-codegen-schema/src/SchemaCRUDPlugin.ts

```diff
@@ -75,7 +75,8 @@ export class SchemaCRUDPlugin extends GraphbackPlugin {
     const resolvers: IResolvers = {
       Query: {},
       Mutation: {},
-      Subscription: {}
+      Subscription: {},
+      JSON: GraphbackJSON
```

Comment:
This will clash with people having JSON. We need to introduce our own namespace scalar - not 100% sure but in all client apps we have JSON already defined

Reply:
Yeah, thing is that the name of the scalar comes from `graphql-type-json`, so even if we changed the resolver name, the scalar being used is still the same and it affects things like FilterInput generation.

There are ways around it, by adapting whats in https://github.com/aerogear/graphback/pull/1620/files#diff-f3d890fe21a2f3c11be271e48b1450f0R12 to not check the scalar name, but it seemed a bit hacky to me.